osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -265,6 +276,18 @@ protected virtual void BindProcessor(ScoreProcessor processor)
             ComboCounter?.Current.BindTo(processor.Combo);
             HealthDisplay?.Current.BindTo(processor.Health);
 
+            if (LeftHitErrorDisplay != null)
+            {
+                processor.NewJudgement += LeftHitErrorDisplay.OnNewJudgement;
+                LeftHitErrorDisplay.HitWindows = processor.CreateHitWindows();
+            }
+
+            if (RightHitErrorDisplay != null)
+            {
+                processor.NewJudgement += RightHitErrorDisplay.OnNewJudgement;
+                RightHitErrorDisplay.HitWindows = processor.CreateHitWindows();
```

Comment:
why is hit windows created twice? 
`var hitWindows = processor.CreateHitWindows()`